pkg/processor/build/runtime/java/test/java_test.go

```diff
@@ -104,29 +102,14 @@ func (suite *TestSuite) TestBuildWithJar() {
 	createFunctionOptions := suite.GetDeployOptions("reverser",
 		suite.GetFunctionPath(suite.GetTestFunctionsDir(), "java", "reverser-jar", "reverser.jar"))
 
-	createFunctionOptions.FunctionConfig.Spec.Runtime = "java"
 	createFunctionOptions.FunctionConfig.Spec.Handler = "Reverser"
-
 	suite.DeployFunctionAndRequest(createFunctionOptions,
 		&httpsuite.Request{
 			RequestBody:          "abcd",
 			ExpectedResponseBody: "dcba",
 		})
 }
 
-func (suite *TestSuite) TestBuildWithJarFromURL() {
```

Comment:
Why remove the test?

Reply:
duplicate test

Reply:
@liranbg  `DeployFunctionAndRequest` == `DeployFunctionFromURL` ?

Reply:
oh, wow, completely miss that one. will undo